lib/trento/application/integration/checks/adapter/gen.ex

```diff
@@ -3,7 +3,7 @@ defmodule Trento.Integration.Checks.Gen do
   Behaviour of a runner adapter.
   """
 
-  alias Trento.Integration.Checks.Models.FlatCatalog
+  alias Trento.Integration.Checks.FlatCatalogDto
```

Comment:
What about `Trento.Integration.Checks.Dto.FlatCatalog`?

Not a strong opinion tho :smile:

Reply:
if you do this when you alias you loose the Dto information, making the code a bit funky